lms/djangoapps/course_api/blocks/tests/test_api.py

```diff
@@ -234,7 +234,7 @@ def test_query_counts_cached(self, store_type, with_storage_backing):
             self._get_blocks(
                 course,
                 expected_mongo_queries=0,
-                expected_sql_queries=11 if with_storage_backing else 10,
+                expected_sql_queries=13 if with_storage_backing else 12,
```

Comment:
So all of these adding in 2 queries is because the ExperimentWaffleFlag used to short circuit where CourseWaffleFlag does not. The 2 queries are checking if there is an override and then the value of the flag.